rfc/index.md

```diff
@@ -94,7 +77,7 @@ beforehand, to ascertain that the RFC may be desirable; having a consistent
 impact on the project requires concerted effort toward consensus-building.
 
 The most common preparations for writing and submitting an RFC include talking
-the idea over on our [Tremor chat](https://join.slack.com/t/tremor-debs/shared_invite/enQtOTMxNzY3NDg0MjI2LTQ4MTU4NjlkZDk0MmJmNmIwYjU0ZDc1OTNjMGRmNzUwZTdlZGVkMWFmNGFkZTAwOWJlYjlkMDZkNGNiMjQ2NzI), and occasionally posting "pre-RFCs" on the
+the idea over on our [Tremor chat](https://chat.tremor.rs), and occasionally posting "pre-RFCs" on the
```

Comment:
:+1: good catch

Reply:
:D ja I found a few more too, so much easier now that everything is in one place